src/index.ts

```diff
@@ -11,7 +11,6 @@ export {
   encrypt,
   decrypt,
 } from './crypto'
-export { getRandomValues } from './crypto/utils'
```

Comment:
@nakajima i looked for this being imported from the content types [repo](https://github.com/xmtp/xmtp-js-content-types), but couldn't find anything.

Reply:
I guess technically this is a breaking change. Even though I highly doubt people are widely importing this, we might have to bring it back to make this a non-major release. We can mark it as deprecated and remove it altogether in v9.